packages/babel-plugin-extract-messages/test/index.ts

```diff
@@ -38,15 +37,16 @@ const transformCode = (code: string, filename = 'test-case.js', rootDir=".") =>
       filename,
       plugins: [
         "@babel/plugin-syntax-jsx",
-        "macros",
+        ["macros", {
+          lingui: { extract: true },
+        },],
```

Comment:
Instead of fighting with `process.NODE_ENV == production` just explicitly pass option to the babel macros. When this option is set macro would not strip non essentials props.